src/messages/gateway.rs

```diff
@@ -161,6 +161,7 @@ pub struct NewWorkerMessage {
 pub struct LostWorkerMessage {
     pub worker_id: WorkerId,
     pub running_tasks: Vec<TaskId>,
+    pub message: String, // Details about reason why worker is lost
```

Comment:
It would maybe be better if this was an enum (e.g. for nicer worker error state formatting in HQ tables).

Reply:
Good point